src/client/dfuse/ops/listxattr.c

```diff
@@ -30,17 +32,26 @@ dfuse_cb_listxattr(fuse_req_t req, struct dfuse_inode_entry *inode,
 
 	D_ALLOC(value, out_size);
 	if (!value)
-		D_GOTO(err, rc = ENOMEM);
+		D_GOTO(free, rc = ENOMEM);
```

Comment:
So if we fail to alloc value we goto free, which does `D_FREE(value);`. This won't hurt, but why?